cocmd/cocmd
# Bugfix for Issue #73 - Uninstall Command bug

## Overview
This pull request introduces a fix for issue #73, where the `cocmd uninstall` command did not remove the associated package files from the `~/.cocmd/runtime/` directory. The fix ensures that the package directory is deleted from the filesystem after the package is uninstalled from cocmd's internal management system.

## Changes
- Modified the `remove_package` function to include a filesystem deletion call.
- Added error handling to manage potential deletion failures and provide feedback.

## How to Test
1. Run `cocmd install zsh`.
2. Run `cocmd uninstall zsh`.
3. Verify that `ls -ltr ~/.cocmd/runtime/zsh` indicates the directory no longer exists.

## Impact
This change ensures that uninstalling packages via cocmd is a clean operation, leaving no unnecessary files or directories behind, thus maintaining system hygiene.

Closes #73
